test.sh

```diff
@@ -1,15 +1,15 @@
-rm -rf /tmp/sqlitefdw_test*.db
-rm -rf /tmp/*.data
-rm -rf /tmp/sqlitefdw_test*.db
-cp -a sql/init_data/*.data /tmp/
+rm -rf /tmp/sqlitefdw_test*.db;
```

Comment:
Could you explain why `;` character added?

Reply:
This have added as command separator. Sometimes during email transfer line separator in transferred text files can be changed from `\n` to `\r` or something similar. This tiny change allows to ignore this problems.

Reply:
Thank you.   
Please help me to resolve this thread.